lblod/frontend-gelinkt-notuleren
# GN-5275: RMW mandatee tables - move bekrachtigtAanstellingVan predicate to 'eedafleggingen' table

### Overview
This PR contains two adjustments:
- Add `mandaat:bekrachtigtAanstellingVan` predicate to mandatees in 'Eedafleggingen' table (`IVRMW2-LBM-5-eed-leden`)
- Remove `mandaat:bekrachtigtAanstellingVan` predicate from 'Verkiezingen' table (`IVRMW2-LBM-3-verkiezing-leden`)

##### connected issues and PRs:
[GN-5275](https://binnenland.atlassian.net/browse/GN-5275)

### How to test/reproduce
- Start the frontend
- Login as an OCMW
- Create/open an IV
- Sync it
- The second AP should contain a filled-in 'eedafleggingen' table. This table should now contain the `mandaat:bekrachtigtAanstellingVan`  to the decision
- The 'verkiezingen' table (if used) should no longer contain any `mandaat:bekrachtigtAanstellingVan` predicates


### Checks PR readiness
